web/app/settings/settings.screensaver.html

```diff
@@ -0,0 +1,192 @@
+                                required>
+                            <span>
+                                <em translate-keep-content translate="screensaver.settings.idletimeout.info">Seconds of being idle until screensaver kicks in</em>
+                            </span>
+                        </div>
+                    </li>
+
+                </ul>
+            </div>
+            <div class="scrollable box">
+                <h4 translate="{{::translations.headingonstart}}"></h4>
+                <ul class="settings-outer">
+                    <li>
+                        <label translate-keep-content translate="screensaver.settings.common.perform" for="ss_onStarType">Perform: </label>
+                        <div>
+                            <select id="ss_onStarType" class="d-block" name='onStarType' ng-model="config.onStart.type" ng-change="validate()">
+                                <option translate-keep-content translate="screensaver.settings.common.gotodashboard" value='gotodashboard'>Go to dashboard</option>
+                                <option translate-keep-content translate="screensaver.settings.onstarttype.options.slideshow" value='slideshow'>Dashboard Slideshow</option>
+                            </select>
+                            <span class='helpinfo'>
+                                <a ng-click='onStartTypeHelp = !onStartTypeHelp'>
+                                    <em>
+                                        <span translate-keep-content translate="screensaver.settings.onstarttype.info">What to do when screensaver starts</span>
+                                        <span class='text-info glyphicon glyphicon-question-sign'></span>
+                                    </em>
+                                </a>
+                                <ul ng-if="onStartTypeHelp">
+                                    <li translate-keep-content translate="screensaver.settings.onstarttype.help.line1">Dashboard Slideshow - rotates between dashboards.</li>
+                                    <li translate-keep-content translate="screensaver.settings.onstarttype.help.line2">Go to dashboard - activates the selected dashboard.</li>
+                                </ul>
+                            </span>
+                        </div>
+                    </li>
+                    <section ng-if="config.onStart.type==='slideshow'">
+                        <li>
+                            <label translate-keep-content translate="screensaver.settings.slideshow.dashboardtimeout.label" for="ss_dashboardTimeout">Slideshow Interval: </label>
+                            <div>
+                                <input class="d-block" type="number" ng-min="1" name="dashboardTimeout" id="ss_dashboardTimeout" ng-model="config.slideshowIntervalSec"
+                                    ng-change="validate()" ng-model-options="{allowInvalid: true}" required>
+                                <span>
```

Comment:
Wonder whether this is really needed. Seems obvious, adds translation work and could be handled in the docs instead.